code/game/gamemodes/miniantags/bot_swarm/swarmer.dm

```diff
@@ -358,6 +358,14 @@
 	S << "<span class='warning'>This object is receiving unactivated swarmer shells to help us. Aborting.</span>"
 	return FALSE
 
+/obj/stucture/lattice/catwalk/swarmer_act(mob/living/simple_animal/hostile/swarmer/S)
+	var/turf/here = get_turf(src)
+	for(var/obj/structure/cable/A in here.contents)
```

Comment:
you can do 
`. = ..()
for(var/A in here.contents)
    var/obj/structure/cable/C = A
    if(istype(C))
        return FALSE`

Or you can just remove swarmers eating catwalks.